<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fileManage</class>
 <widget class="QDialog" name="fileManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>File Transfer</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="pbls">
            <property name="text">
             <string>List File</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbTransfer">
            <property name="text">
             <string>Transfer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbRemove">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Plot Data</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" rowspan="2">
        <widget class="QTreeView" name="treeView"/>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QListView" name="listView"/>
       </item>
       <item row="0" column="2">
        <widget class="QTextEdit" name="textEdit">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>240</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="pbPlot">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Plot</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="leStart">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>10</y>
            <width>113</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="leEnd">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>40</y>
            <width>113</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>1000</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pbShowChart">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>70</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Chart</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pbClearChart">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>70</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Parse Data</string>
      </attribute>
      <widget class="QPushButton" name="pbParse">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>40</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Parse</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="teParseSrc">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>391</width>
         <height>291</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="teParseDest">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>10</y>
         <width>251</width>
         <height>301</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
